src/capture/capture_video.cpp

```diff
@@ -443,27 +444,29 @@ void capture_video_add_frame(const RenderedImage& image, const float frames_per_
 	const auto& src = image.params;
 	assert(src.width <= SCALER_MAXWIDTH);
 
-	// Pixel-doubling is never "baked-in", so we always write the frames
-	// non-pixel-doubled. The only exception is composite modes; these are
-	// rendered at 2x width and we want to write them as such (we need enough
-	// horizontal resolution to properly represent the composite artifacts).
-	const auto video_width = src.width;
+	// To reconstruct the raw image, we must skip every second row when
+	// dealing with "baked-in" double scanning.
+	const auto output_width = check_cast<uint16_t>(
+	        src.width / (src.rendered_pixel_doubling ? 2 : 1));
 
-	// We always write non-double-scanned frames in raw video capture mode
-	const auto video_height = src.video_mode.height;
+	// To reconstruct the raw image, we must skip every second pixel
+	// when dealing with "baked-in" pixel doubling.
+	const auto output_height = check_cast<uint16_t>(
+	        src.height / (src.rendered_double_scan ? 2 : 1));
```

Comment:
Suggest `raw_width`, `raw_height` to use matching terminology with concepts the comments description (and match with the same names in the `compress_frame` function).

Reply:
Done.